.github/data/distros.yml

```diff
@@ -17,6 +17,35 @@ arch_order:  # sort order for per-architecture jobs in CI
   - arm64
   - aarch64
 include:
+# This matrix element builds packages for EL, Alma & Rocky
+  - &rocky
+    distro: rockylinux
+    version: "9"
+    support_type: Core
+    notes: ''
+    jsonc_removal: |
+      dnf remove -y json-c-devel
+    eol_check: true
+    packages: &rocky_packages
+      type: rpm
+      repo_distro: el/9
+      alt_links:
+        - el/9Server
+        - el/9Client
+      arches:
+        - x86_64
+        - aarch64
+    test:
+      ebpf-core: true
+  - <<: *rocky
+    version: "8"
+    packages:
+      <<: *rocky_packages
+      repo_distro: el/8
+      alt_links:
+        - el/8Server
+        - el/8Client
+
```

Comment:
By ‘sorted’, I meant sorted alphabetically. Rocky should be down between Fedora and openSUSE.

Reply:
I’ve so far been sorting alphabetically by the value in the `distro` key, since that ends up translating tot hings being sorted by what most people are likely to be looking for.